etrago/appl.py

```diff
@@ -415,6 +414,13 @@ def run_etrago(args, json_path):
         a new busmap must be calculated. False or path to the busmap in csv
         format should be given.
         Default: False
+        * "interest_area": False, list, string
+        Area of especial interest that will be not clustered. It is by
+        default set to false. When an interest_area is provided, the given
+        value for n_clusters_AC will mean the total of AC buses outside the
```

Comment:
The n_clusters_AC does not apply here. Also needs to be indented in order to be rendered correctly with Sphinx.